Modules/Files/src/Files.ConsumerApi/Controllers/FilesController.cs

```diff
@@ -38,7 +39,9 @@ public FilesController(IMediator mediator, IOptions<ApplicationOptions> options)
     [ProducesError(StatusCodes.Status400BadRequest)]
     public async Task<IActionResult> UploadFile([FromForm] CreateFileDTO dto, CancellationToken cancellationToken)
     {
-        //TODO model state is dto valid
+        var validationResult = await new CreateFileDTOValidator().ValidateAsync(dto, cancellationToken);
+        if (!validationResult.IsValid)
+            return BadRequest(validationResult.Errors);
```

Comment:
That's absolutely not what we want errors returned by our APIs to look like.

Reply:
Updated to throw a `ValidationException` as the Mediatr Pipeline Behavior does.

Reply:
Did you test it?

Reply:
![image](https://github.com/user-attachments/assets/54176b7a-778d-4477-8ec0-4bcca0590869)

Reply:
The error in your screenshot does not come from the `CreateFileDTOValidator`.

Reply:
Uhm, it does, from the first rule on the `Owner` field which validates the `IdentityAddress`